buku.py

```diff
@@ -1492,6 +1492,31 @@ def get_page_title(resp):
         return htmlparser.parsed_title
 
 
+def create_poolmanager(url):
```

Comment:
Not so many checks please! And let's support only https proxies. I'm looking at something clean like [this](https://github.com/jarun/googler/pull/141/files).